h/templates/client/annotation.html

```diff
@@ -221,6 +221,18 @@
               class="share-dialog-target__icon h-icon-mail"></a>
           </span>
           <input class="share-dialog-link" type="text" value="{{vm.annotationURI}}" readonly>
+          <span class="share-dialog-msg" ng-if="vm.group() && !vm.group().public">
```

Comment:
If an annotation is both in a group and private, both messages show up. If an annotation is private, we should only show the second (Only me) one.